test/test_interface.h

```diff
@@ -41,19 +41,23 @@ class TestInterface {
   void VoidMethodTestEnumArg(const std::string& string);
 
   // Attributes
-  double doubleNumber;
-  double readonlyDoubleNumber;
-  static double staticDoubleNumber;
-  double GetDoubleAttribute();
   void ReadonlyAssignTest(double number);
   static double StaticTest(double number);
+  double GetDoubleNumber();
+  void SetDoubleNumber(double number);
+  double GetReadonlyDoubleNumber();
+  static double GetStaticDoubleNumber();
+  static void SetStaticDoubleNumber(double number);
 
  private:
   // FIXME(zino): Currently, we should set this variable in each methods. It's
   // not elegance way. We should find a way to get function name and signature
   // automatically. (I tried __FUNCTION__ and __PRETTY_FUNCTION__ but they are
   // dependent on each platform.
   static std::string last_call_info_;
+  double doubleNumber;
```

Comment:
We follows Chromium C++ style(including Google C++ style):
https://google.github.io/styleguide/cppguide.html#Variable_Names (Class Data Members)

So, could you please append a trailing underscore?

Reply:
I forgot to change it. I'll fix it.

Reply:
@byeolbit, Hmm, you had to append a trailing underscore '_' :)